mathquill4quill.js

```diff
@@ -97,6 +97,15 @@ window.mathquill4quill = function(dependencies) {
       }
     }
 
+    function removeItemFromHistoryList(array, index) {
+      array.splice(index, 1)
```

Comment:
Nit: apply `./node_modules/.bin/prettier --write "**/*.js"` so that CI passes.

```suggestion
      array.splice(index, 1);
```